mavros_extras/src/plugins/mocap_fake_gps.cpp

```diff
@@ -88,52 +89,51 @@ class MocapFakeGPSPlugin : public plugin::PluginBase
 	void mocap_tf_d_cb(const geometry_msgs::TransformStamped::ConstPtr &trans)
 	{
 		// Throttle incoming messages to 5hz
-		if ((ros::Time::now() - last_pos_time).toSec() < gps_period) 
+		if ((ros::Time::now() - last_pos_time).toSec() < gps_period)
 		{
 			return;
 		}
 		last_pos_time = ros::Time::now();
 
-		const double lat_zurich = 47.3667 * M_PI / 180 ;  // rad
-		const double lon_zurich = 8.5500 * M_PI / 180;  // rad
-		const float earth_radius = 6371000;  // m
+		const double lat_zurich = 47.3667 * M_PI / 180;	// rad
+		const double lon_zurich = 8.5500 * M_PI / 180;	// rad
+		const float earth_radius = 6371000;		// m
 
 		Eigen::Quaterniond q_enu;
 
 		tf::quaternionMsgToEigen(trans->transform.rotation, q_enu);
 		auto q = ftf::transform_orientation_enu_ned(ftf::transform_orientation_baselink_aircraft(q_enu));
 
-		auto position = ftf::transform_frame_enu_ned(
-		Eigen::Vector3d(
-		trans->transform.translation.x,
-		trans->transform.translation.y,
-		trans->transform.translation.z));
+		auto position = ftf::transform_frame_enu_ned(Eigen::Vector3d(
+						trans->transform.translation.x,
+						trans->transform.translation.y,
+						trans->transform.translation.z));
 
 		double north = position.x();
 		double east = position.y();
 		double n_rad = north / earth_radius;
 		double e_rad = east / earth_radius;
-		double z = -position.z(); //[m]
+		double z = -position.z();	//[m]
 		double c = sqrt(n_rad * n_rad + e_rad * e_rad);
 		double sin_c = sin(c);
 		double cos_c = cos(c);
 		double lat_rad;
 		double lon_rad;
-		if (c != 0.0) 
+		if (c != 0.0)
 		{
 			lat_rad = asin(cos_c * sin(lat_zurich) + (n_rad * sin_c * cos(lat_zurich)) / c);
-		  lon_rad = (lon_zurich + atan2(e_rad * sin_c, c * cos(lat_zurich) * cos_c - n_rad * sin(lat_zurich) * sin_c));
-		} 
-		else 
+			lon_rad = (lon_zurich + atan2(e_rad * sin_c, c * cos(lat_zurich) * cos_c - n_rad * sin(lat_zurich) * sin_c));
+		}
+		else
 		{
-		 	lat_rad = lat_zurich;
-		  lon_rad = lon_zurich;
+			lat_rad = lat_zurich;
+			lon_rad = lon_zurich;
 		}
```

Comment:
Hmm, i think that uncrustify should fix if block style.

I prefer that style:
```
if (true) {
    do_true();
}
else {
    do_false();
}
```

Or linux style:
```
if (abc) {
} else {
}
```